config/standard/tasks.js

```diff
@@ -0,0 +1,226 @@
+    } ],
+    priority: {
+      level: 'high',
+      label: [ { locale: 'en', content: 'Danger Signs' } ],
+    },
+    resolvedIf: function(c, r, event, dueDate) {
+      // Only resolved with PNC report received from nurse in time window or a newer pregnancy
+      return r.reported_date < getNewestDeliveryTimestamp(c) ||
+             r.reported_date < getNewestPregnancyTimestamp(c) ||
+             isFormFromArraySubmittedInWindow(c.reports, 'postnatal_visit',
+                 Utils.addDate(dueDate, -event.start).getTime(),
+                 Utils.addDate(dueDate,  event.end+1).getTime());
+    },
+  },
+
+  // PNC TASK 3: Assign a missing visit schedule to last SMS of each group
+  // Associate tasks to the last message of each group. Be mindful of overflow when peaking ahead!
+  {
+    name: 'postnatal-missing-visit',
+    icon: 'mother-child',
+    title: [ { locale:'en', content:'Missing postnatal visit' } ],
+    appliesToForms: [ 'D', 'delivery' ],
+    appliesIf: function() { return isCoveredByUseCase(c.contact, 'pnc'); },
+    appliesToScheduledTaskIf: function(r, i) {
+      return i+1 >= r.scheduled_tasks.length ||
+          r.scheduled_tasks[i].group !== r.scheduled_tasks[i+1].group;
+    },
+    priority: function(c, r) {
+      if(isHomeBirth(r)) {
+        return {
+          level: 'high',
+          label: [ { locale:'en', content:'Home Birth' } ],
+        };
+      }
+    },
+    actions: [ { form:'postnatal_visit' } ],
+    events: [ {
+      id: 'postnatal-missing-visit',
+      days: 1, start: 0, end: 3,
+    } ],
+    resolvedIf: function(c, r, event, dueDate, i) {
+      // Resolved if the scheduled SMS that generated the task is cleared,
+      //          if there a visit report received in time window or a newer pregnancy
+      return r.scheduled_tasks[i].state === 'cleared' ||
+          r.reported_date < getNewestDeliveryTimestamp(c) ||
+          r.reported_date < getNewestPregnancyTimestamp(c) ||
+          isFormFromArraySubmittedInWindow(c.reports, postnatalForms,
+              Utils.addDate(dueDate, -event.start).getTime(),
+              Utils.addDate(dueDate,  event.end+1).getTime());
+    },
+  },
+
+  // IMM Task based on Child Health monthly SMS
+  // Assign task to specific age in months corresponding to the group number
+  {
+    name: 'immunization-missing-visit',
+    icon: 'immunization',
+    title: [ { locale:'en', content:'Missing immunization visit' } ],
+    appliesToForms: [ 'CW', 'child_health_registration' ],
+    appliesIf: function() { return isCoveredByUseCase(c.contact, 'imm'); },
+    appliesToScheduledTaskIf: function(r, i) { return immunizationMonths.indexOf(r.scheduled_tasks[i].group) !== -1; },
+    actions: [ { form:'immunization_visit' } ],
+    events: [ {
+      id: 'immunization-missing-visit',
+      days:21, start:7, end:13,
+      // set dueDate explicitly because default dueDate is +28 days and not sure why
```

Comment:
This is due to https://github.com/medic/medic-conf/issues/73. This event's `dueDate` can be removed once fixed in medic-conf